docs/send-signed-transactions.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Send Signed Transaction with the Flow CLI
+sidebar_title: Send Signed Transaction
+description: How to send signed Flow transaction from the command line
+---
+
+The Flow CLI provides a command to send signed transactions to
+any Flow Access API.
+
+Use this functionality in the following order:
+1. Use this command to build the transaction
+2. Use the sign command to sign with all accounts specified in the build process
+3. Use send signed command to submit the signed transaction to the network.
+
+```shell
+flow transactions send-signed <signed transaction filename>
+```
+
+## Example Usage
+
+```shell
+> flow transactions send-signed ./signed.rlp
```

Comment:
This example shows how to load from an RLP file, but the `build` example doesn't show how to save to an RLP file. Maybe add that so that users aren't confused